src/ui/components/shared/Viewport.tsx

```diff
@@ -13,7 +13,7 @@ function FullViewportWrapper({
   return (
     <main
       className={classNames(
-        "w-full fixed h-full z-50 flex items-center justify-center",
+        "w-full fixed h-full z-50 flex items-center justify-center bg-chrome",
```

Comment:
What was the reason for adding `bg-chrome` here?

Reply:
Yeah, so it's no longer possible to get into this state (because the modal behind will close if you pop an error), but previously it was possible to have both the modal and the error open, and because the background to the error is translucent, and so is the background to the modal (but with the modal body sitting on top) you end up with this:

![CleanShot 2022-01-18 at 14 32 23@2x](https://user-images.githubusercontent.com/5903784/150029359-ccfdefaf-3da0-4188-a754-6c1f5b42e4cf.png)